README.md

```diff
@@ -162,27 +162,37 @@ const resolvers = {
 
 # Typescript
 
-As the library cannot know the structure of your user object you need to add this to the global scope by writing your own "somename.d.ts" you can adapt the following template with whatever user variables that you want have added to the PassportRequest:
+As the library cannot know the structure of your user object you need to add this to the global scope by writing your own "somename.d.ts" you can add how your context definition should look like:
 
 ```ts
-declare module "graphql-passport" {
-  interface PassportRequest {
-    user?: {
-      username: string;
-    };
+import { Request as ExpressRequest } from 'express';
+import { PassportSubscriptionContext, PassportContext } from 'graphql-passport';
+import { MyUserObject } = 'some_user_file';
+
+export interface MyContext extends PassportContext<MyUserObject, ExpressRequest>{
+  dataSources: {
+    myOtherAPI: { ... }
+  }
+}
+
+export interface MySubscriptionContext extends PassportSubscriptionContext<MyUserObject, ExpressRequest>{
+  dataSources: {
+    myOtherAPI: { ... }
   }
 }
 ```
 
 In your GraphQL functions you can the reference the passport context using:
 
 ```ts
+import { MyContext } from 'somename';
+
 const something = async (
```

Comment:
`something` is a resolver, right? Maybe find a better name like `username`?

Reply:
Fixed